### PR TITLE
vg: introduce MultiCanvas, a tee for canvases

### PR DESCRIPTION
**Pull request by sbinet**
Please take a look.

### CODE REVIEW

File: vg/tee.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+type teeCanvas struct {
+	cs []Canvas
+}
+
+// SetLineWidth sets the width of stroked paths.
+// If the width is not positive then stroked lines
+// are not drawn.
+//
+// The initial line width is 1 point.
```
Comment (kortschak) [inline vg/tee.go:28]: For these comments on "The initial ...", I would rather say that the initial state depends on the underlying `Canvas`.

File: vg/tee.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+// The initial color is black.  If SetColor is
+// called with a nil color then black is used.
+func (tee teeCanvas) SetColor(c color.Color) {
+	for _, canvas := range tee.cs {
+		canvas.SetColor(c)
+	}
+}
+
+// Rotate applies a rotation transform to the
+// context.  The parameter is specified in
```
Comment (kortschak) [inline vg/tee.go:63]: Reduce double space to single.

File: vg/tee.go [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+// DrawImage draws the image, scaled to fit
+// the destination rectangle.
+func (tee teeCanvas) DrawImage(rect Rectangle, img image.Image) {
+	for _, c := range tee.cs {
+		c.DrawImage(rect, img)
+	}
+}
+
+var (
```
Comment (kortschak) [inline vg/tee.go:137]:
```
var _ Canvas = (*teeCanvas)(nil)
```

File: vg/tee.go (hunk 1 above)
Comment (sbinet) [reply vg/tee.go:28]: I removed those.

File: vg/tee.go (hunk 2 above)
Comment (sbinet) [reply vg/tee.go]: (also removed from the original `vg.Canvas` documentation)

**Comment by sbinet** [comment]
PTAL

**Pull request merged**